src/main.js

```diff
@@ -70,8 +74,18 @@ categoryBox.addEventListener('click', function (event) {
 });
 
 //functions
+function displayHomePage() {
```

Comment:
Nice work changing those page views back and forth at the click of a button!

Reply:
Love that you're organizing code with sections labelled with sections.  

As a general note, if you don't need a global variable, I would scope those inside of the functions.

For example, if you only call on a specific button in one function, add that query selector in that function itself. You can still have it set to "buttonName" inside for readibility, but the closer the definition of a variable is to where it's being used, the better.